ra/ra.go

```diff
@@ -55,6 +55,16 @@ import (
 	"github.com/letsencrypt/boulder/web"
 )
 
+const (
```

Comment:
Given that you defined new constants in the ratelimit package for the names of various rate limits which are only used for the purpose of being metric labels, why not have these be defined in the ratelimit package too?

Especially because it seems likely that the redis version of these metrics will be inside the ratelimit package, rather than in the wfe, having these constants there already will be useful.

Reply:
Oh, nice catch! Done.